### ISSUE TITLE
Cannot run perception_lowcost_vis.sh.

### ISSUE THREAD

**Issue by MistreanuIonutCosmin** [ApolloAuto/apollo#9139]
Hello, and thanks for all the great work done on this framework. 

I am trying to implement perception_lowcost_vis.sh.

I took the content from perception.sh as an example and ended up with:
  
cyber_launch start /apollo/modules/transform/launch/static_transform.launch &
cyber_launch start /apollo/modules/drivers/tools/image_decompress/launch/image_decompress.launch &
cyber_launch start /apollo/modules/perception/production/launch/perception_lowcost.launch &
 
The content of  perception_lowcost.launch is the following:

<cyber>
    <desc>cyber modules list config</desc>
    <version>1.0.0</version>
    <module>
        <name>perception_camera</name>
        <dag_conf>/apollo/modules/perception/production/dag/dag_streaming_perception_camera.dag</dag_conf>
        <process_name>perception</process_name>
        <version>1.0.0</version>
    </module>

    <module>
        <name>perception_lane</name>
        <dag_conf>/apollo/modules/perception/production/dag/dag_streaming_perception_lane.dag</dag_conf>
        <process_name>perception_lane</process_name>
        <version>1.0.0</version>
    </module>
    

    <module>
        <name>perception_traffic_light</name>
        <dag_conf>/apollo/modules/perception/production/dag/dag_streaming_perception_trafficlights.dag</dag_conf>
        <process_name>perception_trafficlights</process_name>
        <version>1.0.0</version>
    </module>

    <module>
        <name>motion_service</name>
        <dag_conf>/apollo/modules/perception/production/dag/dag_motion_service.dag</dag_conf>
        
        <process_name>motion_service</process_name>
        <version>1.0.0</version>
    </module>
</cyber>

When running, I can see the lane detection working but I also got an error: 
[perception_lane]  E0715 15:47:34.780052 12105 debug_info.cc:299] [mainboard] Failed to query camera to ground height and pitch.

I tried debugging it with gdb by running:
$ gdb mainboard
(gdb) run -d /apollo/modules/perception/production/dag/dag_streaming_perception_camera.dag -p perception -s CYBER_DEFAULT

I ended up with the following error which I don't know how to solve:

mainboard: cudnnBuilder2.cpp:1006: nvinfer1::cudnn::Engine* nvinfer1::builder::buildEngine(nvinfer1::CudaEngineBuildConfig&, const nvinfer1::cudnn::HardwareContext&, const nvinfer1::Network&): Assertion `it != tensorScales.end()' failed.

I just started to learn how this framework. Do you have any idea on what I am doing wrong? 

Thanks!

**Comment by MistreanuIonutCosmin** [comment]
Solved the problem by using:
 
  <module>
        <name>perception_camera</name>
        <dag_conf>/apollo/modules/perception/production/dag/dag_streaming_perception_camera.dag</dag_conf>
        <process_name>perception</process_name>
        <version>1.0.0</version>
    </module>

instead of dag_streaming_perception_lane (I found out that the functionality is included in dag_streaming_perception_camera alongside obstacle detection).

I also change in config.pt from "modules/perception/production/data/perception/camera/params"
model_type: "CaffeNet". 
For some reason, the default quantized int8 networks using tensorRT triggers the error above.
